modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/MoveEntries.es.js

```diff
@@ -103,7 +102,12 @@ MoveEntries.STATE = {
 	 * @review
 	 * @type {string}
 	 */
-	selectFolderURL: Config.string().required(),
+	selectFolderURL: {
+		config: {
```

Comment:
I don't think we are supposed to namespace this object with `config` it should just be

```js
selectFolderURL: {
    required: true,
    validator: validateString,
}
```

[metal docs](http://metaljs.com/docs/guides/state.html#configuring_state)

Reply:
It seems you're right @bryceosterhaus. I knew this but in the initial PR I had some discoveries that led me to believe that I needed to namespace it. It works either way, but I'll remove the namespace as it's unnecessary.

Reply:
Handled in https://github.com/liferay-frontend/liferay-portal/pull/853/commits/949c7975125f8b125cd920763521e262631d077f